### PR TITLE
Update GMC manifest changes and misc fixes

### PR DESCRIPTION
**Pull request by lianhao**
## Description

- Update GMC manifest
- Fix global proxy settings in gpt-sovits
- Skip pre-commit for GMC manifest due to upstream format issue
- Adapt latest vllm changes

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Review by Ruoyu-y — APPROVED** [review]
LGTM

File: microservices-connector/config/manifests/llm-uservice_vllm.yaml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    app.kubernetes.io/name: vllm
+    app.kubernetes.io/instance: llm-uservice
+    app.kubernetes.io/version: "0.5"
+    app.kubernetes.io/managed-by: Helm
+data:
+  HF_TOKEN: "insert-your-huggingface-token-here"
+  http_proxy: ""
+  https_proxy: ""
+  no_proxy: ""
+  HABANA_LOGS: "/tmp/habana_logs"
```
Comment (mkbhanda) [inline microservices-connector/config/manifests/llm-uservice_vllm.yaml:21]: is this a gaudi service or just a superfluous here?

File: microservices-connector/config/manifests/tgi_nv.yaml [hunk 2]
```diff
@@ -23,6 +23,9 @@ data:
   HABANA_LOGS: "/tmp/habana_logs"
```
Comment (mkbhanda) [inline microservices-connector/config/manifests/tgi_nv.yaml:23]: is this a gaudi service? what is nv in tgi_nv.yaml?

**Review by mkbhanda — APPROVED** [review]
Why do we  have names that do not match like image: "opea/vllm-hpu:latest" and its yaml file called vllm_gaudi.yaml?

File: microservices-connector/config/manifests/llm-uservice_vllm.yaml (hunk 1 above)
Comment (lianhao) [reply microservices-connector/config/manifests/llm-uservice_vllm.yaml:21]: oops, it's mistake to generate the manifest for CI related settings.

File: microservices-connector/config/manifests/tgi_nv.yaml (hunk 2 above)
Comment (lianhao) [reply microservices-connector/config/manifests/tgi_nv.yaml:23]: good catch. We'll clean up later. Let's get this PR in first to unblock the error of running update_manifest.sh

Comment (lianhao) [reply microservices-connector/config/manifests/tgi_nv.yaml:23]: PR #567 should fix this

**Pull request merged**